chapter_multilayer-perceptrons/kaggle-house-price.md

```diff
@@ -430,6 +463,26 @@ def train(net, train_features, train_labels, test_features, test_labels,
     return train_ls, test_ls
 ```
 
+```{.python .input}
+#@tab paddle
+def train(net, train_features, train_labels, test_features, test_labels,
+          num_epochs, learning_rate, weight_decay, batch_size):
+    train_ls, test_ls = [], []
+    train_iter = d2l.load_array((train_features, train_labels), batch_size)
+    # 这里使用的是Adam优化算法
+    optimizer = paddle.optimizer.Adam(learning_rate=learning_rate, parameters=net.parameters(),weight_decay = weight_decay)
```

Comment:
This line is too long. Please refer to https://book.d2l.ai/user/code.html for more details.